crates/mfem-sys/src/ffi_autocxx.hpp

```diff
@@ -57,13 +61,13 @@ namespace acxx {
     }
 
     std::unique_ptr<FiniteElementSpace> FES_new(
-        Mesh & mesh,
+        Mesh const& mesh,
         FiniteElementCollection const& fec,
         int vdim,
         Ordering::Type ordering)
     {
         return std::make_unique<FiniteElementSpace>(
-            &mesh, &fec, vdim, ordering);
+            const_cast<Mesh*>(&mesh), &fec, vdim, ordering);
```

Comment:
This mutability fudging in currently needed, otherwise there's no way to make `ex1.rs` pass borrow checking...